.github/workflows/compute.yml

```diff
@@ -16,31 +16,25 @@ on:
       ref:
         description: "Ref"
       signature:
-        description: "The kernel signature"
+        description: "Signature sent from the Kernel"
+      command:
+        description: "Command"
 
 jobs:
   compute:
-    name: "plugin name"
+    name: "Pull review"
     runs-on: ubuntu-latest
     permissions: write-all
+    environment: ${{ github.ref == 'refs/heads/main' && 'main' || 'development' }}
```

Comment:
Doesn't seem right to hard code main and development this needs to be dynamic and check against the destination the pull is opened to.

Reply:
I dont quite understand what you mean by a dynamic environment, correct me if i am wrong but isnt an environment in this context used for specifying the where the env variables will be pulled from

Reply:
This thread isn't resolved 

@gentlementlegen rfc

Reply:
This code is correct. It just determines which environment will be pulled from the GitHub Actions, and we currently only have two. We currently don't have one per branch because they have to be manually created, and these don't relate to the kernel nor branch runs.